tools/wptrunner/wptrunner/browsers/chrome.py

```diff
@@ -81,7 +83,8 @@ def executor_kwargs(test_type, server_config, cache_manager, run_info_data,
         chrome_options["args"].extend(kwargs["binary_args"])
 
     # Pass the --headless flag to Chrome if WPT's own --headless flag was set
-    if kwargs["headless"] and "--headless" not in chrome_options["args"]:
+    if ((kwargs["headless"] or test_type == "print-reftest") and
```

Comment:
Why do we need to put the browser into headless mode for print-reftests?

Reply:
Don't know, you tell me ;) But it is apparently a limitation in Chrome that the `printToPDF` CDP endpoint only works in headless.

Reply:
That sounds like a good question to ask a Blink engineer ;) (aiui it's a limitation of Chrome/CDP that the `printToPDF` endpoint only works in headless mode).

Reply:
Ah, I see. Can you add a comment here noting that (maybe linking crbug.com/753118).

cc @foolip on this, since he's in the webdriver space these days and may know which devtools person to mention this to ;)

Reply:
I'm afraid is the first time I've seen crbug.com/753118. While I don't know more than what's on the bug, I've added it to a list of maybe good starter bugs for someone joining soonish.